lib/src/reddit.dart

```diff
@@ -84,51 +101,49 @@ class Reddit {
       Uri redirectUri,
       Uri tokenEndpoint,
       Uri authEndpoint}) {
+    //Loading passed in values into config file.
     final DRAWConfigContext config = new DRAWConfigContext(
         clientId: clientId,
         clientSecret: clientSecret,
         userAgent: userAgent,
         username: username,
         password: password,
-        redirectUri: redirectUri,
-        accessToken: tokenEndpoint,
-        authorizeUri: authEndpoint);
+        redirectUri: redirectUri.toString(),
```

Comment:
Does `toString()` give us the right thing? I seem to remember `toString()` for `Uri` doing something weird, but that could just be my bad memory acting up again.

Reply:
So looked into this, the API doc states the following: "Returns the normalized string representation of the URI."

With regards to normalised, The Unicode Consortium defined normalisation as process for creating a case preserving, standard binary representation of strings.

So It seems to be good on paper, but i'll add extra tests to check this out :)